src/Protocol/Http/Authenticator/JwtAuthenticator.php

```diff
@@ -30,24 +30,61 @@ class JwtAuthenticator implements AuthenticatorInterface
      */
     private $jwt;
 
+    /**
+     * @var string
+     */
+    private $jws = '';
+
+    /**
+     * @var \DateInterval
+     */
+    private $jwsLifetime = null;
+
+    /**
+     * @var \DateTime
+     */
+    private $jwsValidTo = null;
+
     /**
      * Constructor.
      *
-     * @param JwtInterface $jwt
+     * @param JwtInterface  $jwt
+     * @param \DateInterval $jwsLifetime
      */
-    public function __construct(JwtInterface $jwt)
+    public function __construct(JwtInterface $jwt, \DateInterval $jwsLifetime = null)
     {
         $this->jwt = $jwt;
+
+        $this->jwsLifetime = is_null($jwsLifetime) ? new \DateInterval('P1H') : $jwsLifetime;
+        if ($jwsLifetime->invert) {
+            throw new \InvalidArgumentException('JWS lifetime must not be inverted');
+        }
+
+        $this->jwsLifetime = $jwsLifetime;
+    }
+
+    /**
+     * Get jws lifetime
+     *
+     * @return \DateInterval
+    */
+    public function getJwsLifetime() : \DateInterval
```

Comment:
This method is dead, and I think what never be used. We set the lifetime via the constructor, and developer knows what is lifetime he set. Please remove.